apps/site/src/lib/admin/useParticipants.ts

```diff
@@ -3,7 +3,7 @@ import useSWR from "swr";
 
 import { Status, Uid } from "@/lib/admin/useApplicant";
 
-const enum Role {
+export const enum Role {
```

Comment:
Chore: export no longer necessary, but agreed that we should eventually refactor this enum to maybe the user identity so the authorization system can use the enum types.